scripts/build-glean-metadata.py

```diff
@@ -12,8 +12,12 @@
 import yaml
 
 OUTPUT_DIRECTORY = os.path.join("public", "data")
-ANNOTATIONS_URL = "https://mozilla.github.io/glean-annotations/api.json"
-NAMESPACES_URL = "https://raw.githubusercontent.com/mozilla/looker-hub/main/namespaces.yaml"
+ANNOTATIONS_URL = os.getenv(
+    "ANNOTATIONS_URL", "https://mozilla.github.io/glean-annotations/api.json"
```

Comment:
Why do we need to change these to environment variables?

Reply:
It makes testing changes like this a little easier, since you can just do:

```
export ANNOTATIONS_URL=<x>
./scripts/build-glean-metadata.py
```

... without having to modify the source itself. We use a similar pattern in [lookml-generator](https://github.com/mozilla/lookml-generator)